code/api/src/modules/delivery/model.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+// productsHistory
+module.exports = function(sequelize, DataTypes) {
+  let Delivery = sequelize.define('deliveries', {
+    userId: {
+      type: DataTypes.INTEGER
+    },
+    productId: {
+      type: DataTypes.INTEGER
+    },
+    kept: {
+      type: DataTypes.BOOLEAN
+    }
+  })
+
+  Delivery.associate = function(models) {
```

Comment:
Is this what you refer to as a join table?  More generally, is `belongsTo` a way to establish relationships between tables?

Reply:
For sure, @nicolegooden 🤘🏽 joins tables are cool because they allow us to relate a row from one table to a row of another table. This allows us to form relationships between our tables based on how our data should interact with each other. They also serve as the conduit for our many-to-many relationships (for example, a user can have many crates, but a crate can also belong to many users)

Reply:
Exactly @nicolegooden!! The deliveries table is a JOIN table because it joins the users and products table. The way it joins them is by holding userIDs and productsIDs within it that map to those tables. It doesn't have any more information in it - just IDs!

Here's a visual example, the table in the middle in the image below just holds ids that connect the other two tables that actually contain data:

![relational_model](https://user-images.githubusercontent.com/47278429/101853491-81eb5380-3b14-11eb-9c1e-8676433192b3.jpg)

We describe relationships between resources as:

-  **One-to-Many**: For example, the relationship between a Basketball Team and its Players. Many players belong to one team - that's where the `belongsTo` language comes from.
- **Many-to-Many**: For example, the relationships between Recipes and Ingredients. A recipe has many ingredients and ingredients can belong to many recipes. This types of relationships typically need a join table. You may see `hasMany` in some of the associations. This is where it comes from. 

[One to Many](https://backend.turing.io/module2/lessons/one_to_many_relationships)
[Many to Many / Join Tables](https://backend.turing.io/module2/lessons/many_to_many)